content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "Moving cgroup v1 Support into Maintenance Mode in Kubernetes 1.31"
+date: 2024-08-15
+slug: moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31
+author: Harshal Patil
+---
+
+As Kubernetes continues to evolve and adapt to the changing landscape of
+container orchestration, the community has decided to move cgroup v1 support
+into maintenance mode in v1.31. This shift aligns with the broader industry's
+move towards cgroup v2, offering improved functionality, scalability, and
+a more consistent interface. But before we delve into what this means for
+Kubernetes, let's take a step back and understand what cgroups are and their
+significance in Linux and Kubernetes.
+
+## Understanding cgroups
+
+[Control groups](https://man7.org/linux/man-pages/man7/cgroups.7.html), or
+cgroups, are a Linux kernel feature that allows the allocation, prioritization,
+denial, and management of system resources (such as CPU, memory, disk I/O,
+and network bandwidth) among processes. This functionality is crucial for
+maintaining system performance and ensuring that no single process can
+monopolize system resources, which is especially important in multi-tenant
+environments.
+
+There are two versions of cgroups:
+[v1](https://docs.kernel.org/admin-guide/cgroup-v1/index.html) and
+[v2](https://docs.kernel.org/admin-guide/cgroup-v2.html). While cgroup v1
+provided sufficient capabilities for resource management, it had limitations
+that led to the development of cgroup v2. Cgroup v2 offers a more unified
+and consistent interface and better resource control features.
+
+## Cgroups in Kubernetes Kubernetes, a leading container orchestration
```

Comment:
```suggestion
## Cgroups in Kubernetes, a leading container orchestration
```

Reply:
I think you might actually mean:

## Cgroups in Kubernetes

Kubernetes, a leading container orchestration platform

Reply:
Updated, thanks.